File: app/src/main/java/ru/neosvet/vestnewage/list/MenuItem.java
package ru.neosvet.vestnewage.list;

public class MenuItem {
    private String title;
    private int image;
    private boolean select = false;

    public boolean isSelect() {
        return select;
    }

    public void setSelect(boolean select) {
        this.select = select;
    }

    public int getImage() {
        return image;
    }

    public String getTitle() {
        return title;
    }

    public MenuItem(int image, String title) {
        this.image = image;
        this.title = title;

    }
}

File: app/src/main/java/ru/neosvet/vestnewage/data/MenuItem.kt
package ru.neosvet.vestnewage.data

import ru.neosvet.vestnewage.R

class MenuItem(var image: Int, var title: String) {
    var isSelect = false
    var isChecked: Boolean
        get() = image == R.drawable.checkbox_simple
        set(value) {
            image = if (value) R.drawable.checkbox_simple
            else R.drawable.uncheckbox_simple
        }
}